sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/Utility.java

```diff
@@ -31,9 +54,25 @@
  */
 public final class Utility {
     private static final ClientLogger LOGGER = new ClientLogger(Utility.class);
-    // using 4K as default buffer size: https://stackoverflow.com/a/237495/1473510
-    private static final int BYTE_BUFFER_CHUNK_SIZE = 4096;
-    // default time interval for polling
+
+    private static final String DEFAULT_SCOPE = "https://cognitiveservices.azure.com/.default";
+    private static final String FORM_RECOGNIZER_PROPERTIES = "azure-ai-formrecognizer.properties";
+    private static final String NAME = "name";
+    private static final String OCP_APIM_SUBSCRIPTION_KEY = "Ocp-Apim-Subscription-Key";
+    private static final String VERSION = "version";
+
+    private static final ClientOptions DEFAULT_CLIENT_OPTIONS = new ClientOptions();
+    private static final HttpHeaders DEFAULT_HTTP_HEADERS = new HttpHeaders();
+    private static final HttpLogOptions DEFAULT_LOG_OPTIONS = new HttpLogOptions();
```

Comment:
_Not this PR_

Does Form Recognizer have a well-known set of headers and query parameters that are safe for logging? If so, in a separate PR, we should make a static getter on each of the builders which returned a new instance of HttpLogOptions containing these values. Right now, this defaults to the globally known set of safe to log headers and query parameter, take a look at the Search and Storage builders for a detailed explanation of this.

Reply:
Created an issue to address it: https://github.com/Azure/azure-sdk-for-java/issues/20900